Source/DafnyCore/DafnyOptions.cs

```diff
@@ -270,6 +350,15 @@ public enum IncludesModes {
             return true;
           }
 
+        case "compileVerbose": {
```

Comment:
Why aren't this and "useRuntimeLib" handled with legacy bindings?

Reply:
My thought was that we should use RegisterLegacyUi for new options that never existed in the legacy Ui, but for existing legacy options we can just keep the existing code to ensure we don't break anything.

Reply:
I get it now, you're reverting back to the old code from before you extracted `CompileVerboseOption`. Makes sense!